app/models/models.py

```diff
@@ -3,11 +3,11 @@
 
 class QuestionsModels:
 
-    def __init__(self,  title, body, tags, qtn_id):
-        self.qtn_id = qtn_id
-        self.title = title
-        self.tags = tags
-        self.body = body
+    def __init__(self,  title, body, tags, Question_ID):
+        self.Question_ID = Question_ID
```

Comment:
Why use Uuid to generate the ID in the answers, but pass the ID in the questions. Ensure that you use a uniform structure.